test/wrapper.test.ts

```diff
@@ -342,15 +239,19 @@ describe('Tests Native Wrapper', () => {
         // @ts-ignore it is an error but it does not know the type.
         expect(error.message).toMatch('Native is disabled');
       }
-      expect(RNSentry.getStringBytesLength).not.toBeCalled();
       expect(RNSentry.captureEnvelope).not.toBeCalled();
     });
+    test('Encloses message to an object on Android', async () => {
+      // TODO:
+    });
+    test('Do not introduce empty breadcrumbs if undefined before on Android', async () => {
+      // TODO:
+    });
```

Comment:
Are we addressing this in this PR? otherwise I'd remove it.

Reply:
Ohh, thanks, good catch, I definitely want to implement these.